package.json

```diff
@@ -16,6 +16,7 @@
   "dependencies": {
     "@duckduckgo/autofill": "github:duckduckgo/duckduckgo-autofill#5.1.0",
     "@duckduckgo/autoconsent": "github:duckduckgo/autoconsent#v3.0.0",
-    "@duckduckgo/content-scope-scripts": "github:duckduckgo/content-scope-scripts#2.6.0"
+    "@duckduckgo/content-scope-scripts": "github:duckduckgo/content-scope-scripts#2.6.0",
+    "@duckduckgo/privacy-reference-tests": "github:duckduckgo/privacy-reference-tests#main"
```

Comment:
Should this be updated to a version? I see [1.0.0](https://github.com/duckduckgo/privacy-reference-tests/releases/tag/1.0.0) was just released

Reply:
Sorted!